src/ffi/bindings/orx_bindings.ml

```diff
@@ -1277,6 +1285,21 @@ module Bindings (F : Ctypes.FOREIGN) = struct
       Ctypes.getf !@payload T.Time_line_event.Payload.time_stamp
   end
 
+  module Anim_event = struct
+	include T.Anim_event
+	type payload = Payload.t Ctypes.structure Ctypes.ptr
+
+	let compare (a : t) (b : t) = compare a b
+
+	let get_animation (payload : payload) : Animation.t = Ctypes.getf !@payload T.Anim_event.Payload.animation
+	let get_name (payload : payload) : string = Ctypes.getf !@payload T.Anim_event.Payload.name
+	(* TODO *)
+	(* let get_count (payload : payload) : int = Ctypes.getf !@payload T.Anim_event.Payload.count
```

Comment:
Formatting tip - If you comment these line by line rather than all in a block, `ocamlformat` will leave them as separate things rather than smashing them all together with it's comment autoformatting. Should make it a little easier to follow once the union field changes are added.